manifest.json

```diff
@@ -1,8 +1,8 @@
 {
     "spark_version": "2.2.1",
     "default_spark_dist": {
-        "hadoop_version": "2.6",
-        "uri": "https://downloads.mesosphere.com/spark/assets/spark-2.2.1-3-SNAPSHOT-bin-2.6.tgz"
+        "hadoop_version": "2.7",
+        "uri": "https://s3.us-east-2.amazonaws.com/akirilloiv-dev/spark-2.2.1-bin-2.7.3.tgz"
```

Comment:
Before you merge, pls change this url to a `downloads.mesosphere` spark one. You can append `-SNAPSHOT` to denote that it is a dev build and subject to change

Also, is this spark 2.2.1? Wasn't it built from Spark-2.3?

Reply:
The change started from 2.2.1 branch and already merged there, there's a separate PR for Spark 2.3 branch.

Updated versions of Spark distributions are uploaded to `downloads.mesosphere` for both Hadoop 2.6 and 2.7, will wait for the build to succeed and merge the changes after that if it's ok.